doc/Media2.xml

```diff
@@ -196,6 +197,14 @@
         </author>
         <revremark>Clarify signaling rotation in video source configuration.</revremark>
       </revision>
+      <revision>
+        <revnumber>23.12</revnumber>
```

Comment:
change version number and version here.

Reply:
Alson changed version number in Security.xml